static/stylesheets/application.css

```
@@ -1054,6 +1053,15 @@ h6 + .id-link-button {
 .article li p {
     display: inline
 }
+.article li p + ul {
+    margin-top: .9em;
+}
+.article ul li {
+    margin-bottom: 7px;
+}
```

1025行目にも同じ「.article ul li」の指定があり、そちらではマージンを上下に0を設定していました。
なので数値が矛盾してしまうので、次のように修正するといいと思いました。

- 1025目の設定は削除する
- 「.article ul li」の設定を次のように変更する

```suggestion
.article ul li {
    margin: 0 5px 7px 5px;
}
```

数値は、上右下左という意味です。
つまり、上が0、右が5px、下が7px、左が5pxの余白があくと言う意味です。